sql/core/src/main/scala/org/apache/spark/sql/execution/basicPhysicalOperators.scala

```diff
@@ -497,7 +496,9 @@ case class UnionExec(children: Seq[SparkPlan]) extends SparkPlan {
  * if you go from 1000 partitions to 100 partitions, there will not be a shuffle, instead each of
  * the 100 new partitions will claim 10 of the current partitions.
  */
-case class CoalesceExec(numPartitions: Int, child: SparkPlan) extends UnaryExecNode {
+case class CoalesceExec(numPartitions: Int, child: SparkPlan,
+                        partitionCoalescer: Option[PartitionCoalescer]
+                       ) extends UnaryExecNode {
```

Comment:
Do you guys have a .scalafmt.conf that applies all of this?  that should make things cleaner.